Build.fsproj

```diff
@@ -15,4 +14,8 @@
     <Exec Command="dotnet tool restore" />
   </Target>
   <Import Project=".paket\Paket.Restore.targets" />
+  <Target Name="Husky" BeforeTargets="Restore;CollectPackageReferences" Condition="'$(HUSKY)' != 0">
```

Comment:
This ensures that Husky is available after someone has restored the build project, which will be any time they use dotnet run to run the application.